### PR TITLE
Disable caching in Async Storage on Android

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Comment by jinchung** [comment]
I think this should be okay given that most localstorage items we are immediately doing our own version of a memory cache using redux anyway. We may also consider doing this for iOS. Will defer to @skylarbarrera to verify overall performance impact.

**Review by jinchung — APPROVED** [review]
As noted in my comment, may also be helpful in iOS. Approve pending Skylar's approval.

**Comment by skylarbarrera** [comment]
pretty sure this is impacting performance on android, let's keep open for now if you think its worth checking out further

**Pull request merged**
